src/lib/utils/db/authRules.js

```diff
@@ -46,7 +46,23 @@ export function isAdmin(authUser) {
 }
 
 // Helpers for quick-and-easy auth
-export async function allowManagerOrAdmin(db, { params, headers } = {}) {
+export async function allowAdminOnly(db, { headers = undefined } = {}) {
+    let authUser = await verifyJwtAuth(db, headers);
+    if (!authUser) {
+        if (authUser === undefined) {
+            return authTokenRequired();
+        } else {
+            return notAllowed();
+        }
+    }
+    const allowed = isAdmin(authUser);
+    if (!allowed) {
+        return notAllowed();
+    }
+    return { status: 200, authUser };
+}
+
+export async function allowManagerOrAdmin(db, { params = undefined, headers = undefined } = {}) {
```

Comment:
I recommend renaming this function to be `allowProjectManagerOrAdmin()`.  This will make the name more consistent with the other methods.